### PR TITLE
updated README intro

### PR DESCRIPTION
**Pull request by keith-turner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,16 +15,21 @@
 APILyzer: A simple API Analyzer
 ===============================
 
-View this plugin's documentation at: http://revelc.github.io/apilyzer-maven-plugin
+[APILyzer][2] is a maven plugin that detects illegal changes to a maven
```
Comment (ctubbsii) [inline README.md]: So, I kinda want the main link to the plugin documentation very prominent at the top. I'd like to move a lot of the more detailed information here to that site, keeping only the bare minimum here.

For now, I'm fine with the additional details, but I'd still like to keep the URL prominently displayed.

File: README.md [hunk 2]
```diff
@@ -15,16 +15,21 @@
...
+projects Java API.  This accomplished by looking for deviations from the
+following rules.
 
-A simple maven plugin that analyzes types used by declared public API methods
-and generates a report.
+ * Public API members should use other Public API types.
+ * Public API members should use an approved set of types.
```
Comment (ctubbsii) [inline README.md]: trailing whitespace here

File: README.md (hunk 1 above)
Comment (keith-turner) [inline README.md]: Could move most of this documentation to website and have just the 1st sentence in readme

**Comment by keith-turner** [comment]
this is superseded by #9

**Pull request closed without merging**